spot-client/src/spot-remote/app-state/actions.js

```diff
@@ -141,8 +152,6 @@ export function connectToSpotTV(joinCode, shareMode) {
 
             _clearSubscriptions();
 
-            dispatch(clearSpotTVState());
```

Comment:
Why not clear spot tv state?

Reply:
hmm I think I was fighting some issue and have it fixed in another way and this stayed. Will try to undo the change.